<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClinetMainWindow</class>
 <widget class="QMainWindow" name="ClinetMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClinetMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>651</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>视频监控</string>
    </property>
    <widget class="QLabel" name="labVideo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>631</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string>video</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>331</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>控制</string>
    </property>
    <widget class="QComboBox" name="cmbDev">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCamera">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCloseCamera">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>48</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>设备名:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTemperature">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>33</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>温度:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenLight">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCloseLight">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>269</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>269</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenSeneor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCloseSensor">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPicenv">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>环境信息详情</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonback">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>视频回放</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>30</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>灯</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>59</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>环境信息</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
